streamlit_extras/echo_expander/__init__.py

```diff
@@ -90,11 +91,11 @@ def echo_expander(code_location="above", expander=True, label="Show code"):
     except FileNotFoundError as err:
         placeholder.warning("Unable to display code. %s" % err)
 
+
 def example1():
     with echo_expander():
         import streamlit as st
 
-        st.title("Hello Streamlit!")
```

Comment:
Removed this as it would create a sudden title in the middle of the UI which could create confusion - it's not crucial to understanding how function works!